testing/integration/upgrade_test.go

```diff
@@ -424,7 +489,18 @@ func testStandaloneUpgrade(
 		require.NoError(t, err)
 		sourceURI := "file://" + filepath.Dir(srcPkg)
 		t.Logf("setting sourceURI to : %q", sourceURI)
-		upgradeCmdArgs = append(upgradeCmdArgs, "--source-uri", sourceURI, "--skip-verify")
```

Comment:
the skipVerify and source-uri have to go hand in hand when we are testing with the locally built package...
How about setting the conditions to something like:
```
useLocalPackage :=  allowLocalPackage && version_8_7_0.Less(*parsedFromVersion)
skipVerifyFlag := useLocalPackage || skipVerify
```

Or is the skipVerify an override?

If this get too complicated we can always extract all of those as options to be set in the caller function...